src/Symfony/Component/Routing/Route.php

```diff
@@ -65,6 +65,11 @@ class Route implements \Serializable
     private $condition = '';
 
     /**
+     * @var bool
+     */
+    private $unicode = '';
+
```

Comment:
The variable is a boolean, so the default should be `false`, not an empty string.